CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG for ASAB WebUI Shell
 
+## 25.2.1
```

Comment:
@aringocode If you want to create a tag after this is merged, you really need to update the version in `package.json` as well. Otherwise the distribution to the npm will fail.